src/assets/scss/_recordset-table.scss

```diff
@@ -157,21 +157,21 @@
     }
   }
 
+  // Changing the padding for the action button to align with the rest of the columns in the table
   .action-btns {
     text-align: center;
     font-size: 1.2rem;
-    // since we're increasing the size of icons, we should decrease the padding
-    padding: 5px 0 0 0;
-
+    padding: 3px 0 0 0;
     .chaise-btn {
       padding: 0;
       margin: -3px;
       height: auto; //override the default button height
       min-height: 25px; // make sure the button has default height
     }
 
+    // reducing the font-size to make the alignment of view-details better with other action buttons
     .chaise-view-details {
-      font-size: 1.1em;
+      font-size: 1em;
     }
```

Comment:
edit pencil icon was few pixels off from view-details. reducing the font-size looks better in all zoom levels and browsers